src/androidTest/java/com/mixpanel/android/mpmetrics/MixpanelNotificationBuilderTest.java

```diff
@@ -108,6 +119,32 @@ public void testInvalidIcon() {
         verify(builderSpy).setSmallIcon(DEFAULT_ICON_ID);
     }
 
+    public void testExpandedImageUsingValidUrl() throws ImageStore.CantGetImageException{
+        final Intent intent = new Intent();
+        intent.putExtra("mp_message", "MESSAGE");
+        intent.putExtra("mp_imgurl", VALID_URL);
+
+        Bitmap fakeBitmap = getFakeBitmap();
+
+        when(mpPushSpy.getBitmapFromUrl(VALID_URL)).thenReturn(fakeBitmap);
+
+        mpPushSpy.createNotification(intent);
+        verify(mpPushSpy).getBitmapFromUrl(VALID_URL);
+        verify(builderSpy).setStyle(new Notification.BigPictureStyle().bigPicture(fakeBitmap));
+    }
+
+    public void testExpandedImageUsingInvalidUrl() throws ImageStore.CantGetImageException{
+        final Intent intent = new Intent();
+        intent.putExtra("mp_message", "MESSAGE");
+        intent.putExtra("mp_imgurl", "");
```

Comment:
nit: empty strings sometimes have special meaning (generally due to a bad design decision but nevertheless). for this test case let's use something a little more explicit like `intent.putExtra("mp_imgurl", "this is a bad url")`